### PR TITLE
add: start dashboard

### PR DESCRIPTION
**Pull request by jhennecart**
screen
screen

### CODE REVIEW

File: templates/forum/dashboard.haml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+-extends "base.haml"
+
+-load static
+-load student_tags
+-load lesson_tags
```
Comment (fknop) [inline templates/forum/dashboard.haml]: Necessary to load lesson_tags ?

File: templates/forum/dashboard.haml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          %th Auteur
+          %th Compétence
+          %th Date
+
+        {% if threads %}
+        	-for thread in threads
+	          %tr
+	            %td
+	            	%a{:href => "/forum/thread/{{thread.id}}" }>{{thread.title}}
+	            %td {{thread.author}}
```
Comment (fknop) [inline templates/forum/dashboard.haml]: I think it should be thread.author.username

File: templates/forum/dashboard.haml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	            %td {{thread.skills}}
```
Comment (fknop) [inline templates/forum/dashboard.haml]: It won't work like that as skills is a list, it needs a loop as well

**Review by fknop — CHANGES_REQUESTED** [review]

**Comment by fknop** [comment]
Could you also add a screenshot ?

**Review by fknop — DISMISSED** [review]

**Comment by fknop** [comment]
Good for me.

@mlosicki

**Comment by fknop** [comment]
Could you add a Closes #11 in your pull request message

**Review by mlosicki — APPROVED** [review]

**Pull request merged**